save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/AgentService.kt

```diff
@@ -0,0 +1,21 @@
+package org.cqfn.save.orchestrator.service
+
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
```

Comment:
I guess in spring 5 `WebClient` is preferred over RestTemplate

Reply:
Fixed